xla/pjrt/distributed/distributed.cc

```diff
@@ -38,9 +38,14 @@ GetDistributedRuntimeService(std::string address,
 }
 
 std::shared_ptr<DistributedRuntimeClient> GetDistributedRuntimeClient(
-    std::string address, const DistributedRuntimeClient::Options& options) {
-  std::shared_ptr<grpc::Channel> channel = grpc::CreateChannel(
-      address, tsl::GetClientCredentials(kVerifySecureCredentials));
+    std::string address, bool use_compression,
```

Comment:
Super-nit: we already have an options struct, maybe put `use_compression` here?

Reply:
See the comment on that in the PR description! (But do let me know if you feel strongly about it.)